src/pages/workspaces/workspace/Data.js

```diff
@@ -98,7 +98,7 @@ const DataTypeButton = ({ selected, entityName, children, entityCount, iconName
       'aria-current': selected,
       ...props
     }, [
-      activeCrossTableTextFilter !== '' && isEntity ?
+      activeCrossTableTextFilter !== '' && activeCrossTableTextFilter !== undefined && isEntity ?
```

Comment:
To make this a little more concise, maybe one of...

```suggestion
      activeCrossTableTextFilter && isEntity ?
```

```suggestion
      !_.isEmpty(activeCrossTableTextFilter) && isEntity ?
```